doc/api/esm.md

```diff
@@ -560,6 +560,63 @@ node b.mjs # works
 
 <i id="esm_experimental_loaders"></i>
 
+## HTTPS and HTTP imports
+
+> Stability: 1 - Experimental
+
+Importing network based modules using `https:` and `http:` is supported under
+the `--experimental-https-modules` flag. This allows web browser-like imports to
+work in Node.js with a few differences due to application stability and security
+concerns that are different when running in a privileged environment instead of
+a browser sandbox.
+
+### HTTP is limited to loopback addresses
+
+`http:` is vulnerable to man in the middle attacks and is not allowed to be used
+for addresses outside of the IPv4 address 127.0.0.1 and the IPv6 address ::1 .
+Support for `http:` is intended to be used for local development.
+
+### Cookies are never sent to the destination server.
+
+Support for custom headers per location such as `Cookie:` is in development.
+
```

Comment:
May want to say something also about authenticated requests... or lack thereof...

Reply:
I'm torn here since `https:` itself is a secure channel. I think until we have a mitigation implementation it isn't useful to bring up to average docs user.

Reply:
I think it's enough just to say that the `Authorization` header will not be sent and that the import URL specifiers should not contain userinfo segments.

Reply:
reworded